src/main/java/de/tum/in/www1/artemis/domain/User.java

```diff
@@ -238,7 +238,12 @@ public void setRegistrationNumber(String registrationNumber) {
     }
 
     public String getVisibleRegistrationNumber() {
-        return visibleRegistrationNumberTransient;
+        if (visibleRegistrationNumberTransient != null) {
```

Comment:
Please revert this change. It is intended that the registration number is not send with every REST response that includes a user object.
It has to be copied explicitly into the transient field to be sent to the client

Reply:
@LDAP resolved.